_posts/2020-10-05-Evidence_locker_overview.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+category: Architecture
+title: Evidence locker overview
+---
+
+At the core of the [Auditree framework][] is a [Git][] repository known as the
+"evidence locker".  Each Auditree execution environment requires a dedicated
+evidence locker in order to execute.  The Auditree framework can either clone
+an evidence locker from a remote Git repository hosting service (GitHub,
+GitLab, ...) or create a new local locker for you.  Serving as the Auditree
+data store, the evidence locker houses evidence gathered by fetchers, evidence
+generated by checks and/or evidence planted by the [Auditree Plant][] tool.
+
+## Local vs. remote
+
+The reasoning behind our choice of [Git][] as the Auditree data store is
+outlined nicely in our [previous post][] but in a nutshell, a local evidence
+locker works hand-in-hand with a remote evidence locker hosted by any Git
+repository hosting service.  Although it is possible to execute Auditree with
```

Comment:
“...nutshell, an evidence locked uses git as an efficient, reliable, tamper evident data store.”

Reply:
Done in c51751f